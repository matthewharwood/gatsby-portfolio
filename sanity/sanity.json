{
  "root": true,
  "project": {
    "name": "morningharwood"
  },
  "api": {
    "projectId": "42nykw3g",
    "dataset": "production"
  },
  "plugins": [
    "@sanity/base",
    "@sanity/components",
    "@sanity/default-layout",
    "@sanity/default-login",
    "@sanity/desk-tool"
  ],
  "env": {
    "development": {
      "plugins": [
        "@sanity/vision"
      ]
    }
  },
  "parts": [
    {
      "name": "part:@sanity/base/schema",
      "path": "./schemas/schema.js"
    }
  ]
}
