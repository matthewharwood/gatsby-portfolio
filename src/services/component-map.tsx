import React from 'react';
import {PostHeader} from "../components/post-header";
import {Summary} from "../components/summary"
import {Process} from "../components/process"
import {Highlight} from "../components/highlight"
import {HighlightBold} from "../components/highlight-bold"
import {Quote} from "../components/quote";
import {Video} from "../components/video"
import {Challenge} from "../components/challenge"

import { ImageSection } from '../components/image-section';
import { ImageGallery } from '../components/image-gallery';
import {CenteredHeading} from '../components/centered-heading';
import {CodeComponent} from '../components/code-component';

export const componentMap = {
  "postHeader": (props: any, key: number) => (<PostHeader {...props} key={key}/>),
  "summary": (props: any, key: number) => (<Summary {...props} key={key}/>),
  "process": (props: any, key: number) => (<Process {...props} key={key}/>),
  "highlight": (props: any, key: number) => (<Highlight {...props} key={key}/>),
  "highlightBold": (props: any, key: number) => (<HighlightBold {...props} key={key}/>),
  "quote": (props: any, key: number) => (<Quote {...props} key={key}/>),
  "video": (props: any, key: number) => (<Video {...props} key={key}/>),
  "challenge": (props: any, key: number) => (<Challenge {...props} key={key} />),
  "imageSection": (props:any, key:number) => (<ImageSection {...props} key={key}/>),
  "centeredHeading": (props:any, key:number) => (<CenteredHeading {...props} key={key}/>),
  "imageGallery": (props:any, key:number) => (<ImageGallery {...props} key={key}/>),
  "codeComponent": (props:any, key:number) => (<CodeComponent {...props} key={key}/>),
}


export function componentMapFactory() {
  let defaultValueHandler = {
    get: function(target: any, name: any) {
      return target.hasOwnProperty(name)
        ? target[name]
        : (props: any) => console.warn(
          "We are missing a component type: ", props._type, "is not found"
        );
    },
  }

  return new Proxy(componentMap, defaultValueHandler)
}


// export const componentMap2 = {
//   "postHeader": PostHeader,
//   "summary": Summary,
//   "process": Process,
//   "highlight": Highlight,
//   "highlightBold": HighlightBold,
//   "quote": Quote,
//   "video": Video,
//   "challenge": Challenge,
// }
//
// const components = {
//   photo: PhotoStory,
//   video: VideoStory
// };

// function Story(props) {
//   // Correct! JSX type can be a capitalized variable.
//   const SpecificStory = components[props.storyType];
//   return <SpecificStory story={props.story} />;
// }
