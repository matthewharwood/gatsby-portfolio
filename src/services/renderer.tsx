function componentMapFactory() {
  const componentMap = {
    "postHeader": (props: any) => console.log("POSTHEADER", props),
    "summary": (props: any) => console.log("SUMMARY", props),
  }

  let defaultValueHandler = {
    get: function(target: any, name: any) {
      return target.hasOwnProperty(name)
        ? target[name]
        : (props: any) => console.warn(
          "We are missing a component type: ", props._type, "is not found"
        );
    },
  }

  return new Proxy(componentMap, defaultValueHandler)
}

export const pageRenderer = (raw) => {

  const componentMap = componentMapFactory();

  raw.map((d) => [d._type, d]).map(([key, props]) => componentMap[key](props));
  return true
}