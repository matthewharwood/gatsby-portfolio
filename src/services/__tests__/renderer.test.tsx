import React from 'react';
import { componentMapFactory } from '../component-map';
import { SANITY_RAW_CONTENT } from "../__fixtures__/sanity"
import renderer from "react-test-renderer"

function component(index:number){
  const _typeArr = SANITY_RAW_CONTENT._rawContent.map(({_type}) => _type);
  const componentMap = componentMapFactory();
  return componentMap[_typeArr[index]](SANITY_RAW_CONTENT._rawContent[index]);
}

describe('Page Renderer ', () => {
  it('should be _type postHeader', () => {
    const _typeArr = SANITY_RAW_CONTENT._rawContent.map(({_type}) => _type);
    expect(_typeArr[0]).toBe('postHeader');
  });

  it('should render postHeader', () => {
    const Component = component(0);
    const tree = renderer
      .create(Component)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render summary', () => {
    const Component = component(1);
    const tree = renderer
      .create(Component)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render Process', () => {
    const Component = component(2);
    const tree = renderer
      .create(Component)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render Highlight', () => {
    const Component = component(3);
    const tree = renderer
      .create(Component)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should render Highlight 2', () => {
    const Component = component(4);
    const tree = renderer
      .create(Component)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should render Highlight Bold', () => {
    const Component = component(5);
    const tree = renderer
      .create(Component)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render summary 2', () => {
    const Component = component(6);
    const tree = renderer
      .create(Component)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should render summary 3', () => {
    const Component = component(7);
    const tree = renderer
      .create(Component)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render quote', () => {
    const Component = component(8);
    const tree = renderer
      .create(Component)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render Video', () => {
    let Component = component(9);
    const tree = renderer
      .create(Component)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should render Video', () => {
    let Component = component(9);
    const tree = renderer
      .create(Component)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
