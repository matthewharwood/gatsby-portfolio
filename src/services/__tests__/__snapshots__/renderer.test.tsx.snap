// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Renderer  should render Highlight 1`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <section
    className=""
  >
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className=""
        >
          <div
            className=""
            data-baseweb="typo-label1"
          >
            01
          </div>
          <div
            className=""
            data-baseweb="typo-display4"
          >
            Getting Started
          </div>
          <p
            className=""
            data-baseweb="typo-p1"
          >
            Stanley Yeung, a former AKQA designer assigned to Nike Jordan was leaving the company.  Stanley managed the production design work for webpages across Nike Jordan.  Over lunch, Stanley’s creative director jokingly said, “With Stanley leaving, I wish we could create a robot that could continue all his production design work.” We knew we couldn’t make a robot but we saw a potential design challenge.
          </p>
        </div>
        <div
          className=""
        />
      </div>
    </div>
  </section>
</div>
`;

exports[`Page Renderer  should render Highlight 2 1`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <section
    className=""
  >
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className=""
        >
          <div
            className=""
            data-baseweb="typo-label1"
          >
            02
          </div>
          <div
            className=""
            data-baseweb="typo-display4"
          >
            Identifing Pain Points.
          </div>
          <p
            className=""
            data-baseweb="typo-p1"
          >
            alsdjfsd
          </p>
        </div>
        <div
          className=""
        />
      </div>
    </div>
  </section>
</div>
`;

exports[`Page Renderer  should render Highlight Bold 1`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <section
    className=""
  >
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className=""
        >
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="typo-label1"
            >
              03
            </div>
            <div
              className=""
              data-baseweb="typo-display4"
            >
              Proposed Solution
            </div>
            <p
              className=""
              data-baseweb="typo-p1"
            >
              Create a design and authoring tool, that enables, en masse, inhouse or remote Nike teams to collaborate and to rapidly iterate web pages for nike.com.
            </p>
          </div>
          <div
            className=""
          />
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Page Renderer  should render Process 1`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <section
    className=""
  >
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className=""
        >
          <div
            className=""
            data-baseweb="typo-display4"
          >
            Process
          </div>
        </div>
        <div
          className=""
        >
          <ol
            className=""
            data-baseweb="progress-steps"
          >
            <li
              className=""
            >
              <div
                className=""
              >
                <div
                  className=""
                />
              </div>
              <div
                className=""
              />
              <div
                className=""
              >
                <div
                  className=""
                >
                  Getting started
                </div>
                <div
                  className=""
                />
              </div>
            </li>
            <li
              className=""
            >
              <div
                className=""
              />
              <div
                className=""
              />
              <div
                className=""
              >
                <div
                  className=""
                >
                  Identify painpoints
                </div>
                <div
                  className=""
                />
              </div>
            </li>
            <li
              className=""
            >
              <div
                className=""
              />
              <div
                className=""
              />
              <div
                className=""
              >
                <div
                  className=""
                >
                  Proposed Solution
                </div>
                <div
                  className=""
                />
              </div>
            </li>
            <li
              className=""
            >
              <div
                className=""
              />
              <div
                className=""
              />
              <div
                className=""
              >
                <div
                  className=""
                >
                  Execution - Design
                </div>
                <div
                  className=""
                />
              </div>
            </li>
            <li
              className=""
            >
              <div
                className=""
              />
              <div
                className=""
              />
              <div
                className=""
              >
                <div
                  className=""
                >
                  Execution - Development
                </div>
                <div
                  className=""
                />
              </div>
            </li>
            <li
              className=""
            >
              <div
                className=""
              />
              <div
                className=""
              />
              <div
                className=""
              >
                <div
                  className=""
                >
                  Test / Measure / Iterate
                </div>
                <div
                  className=""
                />
              </div>
            </li>
            <li
              className=""
            >
              <div
                className=""
              />
              <div
                className=""
              />
              <div
                className=""
              >
                <div
                  className=""
                >
                  Pitch
                </div>
                <div
                  className=""
                />
              </div>
            </li>
            <li
              className=""
            >
              <div
                className=""
              />
              <div
                className=""
              />
              <div
                className=""
              >
                <div
                  className=""
                >
                  Result
                </div>
                <div
                  className=""
                />
              </div>
            </li>
            <li
              className=""
            >
              <div
                className=""
              />
              <div
                className=""
              >
                <div
                  className=""
                >
                  Retrospective
                </div>
                <div
                  className=""
                />
              </div>
            </li>
          </ol>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Page Renderer  should render Video 1`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <section
    className=""
  >
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className=""
        >
          <div
            className=""
            data-baseweb="aspect-ratio-box"
          >
            <div
              className=""
              data-baseweb="aspect-ratio-box-body"
            >
              <video
                className=""
                onEnded={[Function]}
                onLoadedMetadata={[Function]}
                onTimeUpdate={[Function]}
                poster="https://via.placeholder.com/1280x720?text=Poster+Sample"
                src="https://archive.org/download/NASA-Ultra-High-Definition/Scott4kUhd.mov"
              />
              <div
                className=""
                data-baseweb="progress-bar"
                role="progressbar"
              >
                <div
                  className=""
                >
                  <div
                    className=""
                  />
                </div>
              </div>
            </div>
          </div>
          <button
            className=""
            onClick={[Function]}
          >
            <svg
              height="24"
              style={
                Object {
                  "filter": "drop-shadow( 0px 2px 0.5px rgba(0, 0, 0, .3",
                }
              }
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M8 5v14l11-7z"
                fill="#FFFFFF"
              />
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
            </svg>
          </button>
        </div>
        <div
          className=""
        >
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="typo-label2"
            >
              making a project
            </div>
            <p
              className=""
              data-baseweb="typo-p2"
            >
              We needed a view that would allow authors to create project and view them all at a glance. There were many stakeholders all collaborating.  The idea was to enable any of them to rapidly prototype a page and the best way to do this was to clone and version control another project.
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Page Renderer  should render Video 2`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <section
    className=""
  >
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className=""
        >
          <div
            className=""
            data-baseweb="aspect-ratio-box"
          >
            <div
              className=""
              data-baseweb="aspect-ratio-box-body"
            >
              <video
                className=""
                onEnded={[Function]}
                onLoadedMetadata={[Function]}
                onTimeUpdate={[Function]}
                poster="https://via.placeholder.com/1280x720?text=Poster+Sample"
                src="https://archive.org/download/NASA-Ultra-High-Definition/Scott4kUhd.mov"
              />
              <div
                className=""
                data-baseweb="progress-bar"
                role="progressbar"
              >
                <div
                  className=""
                >
                  <div
                    className=""
                  />
                </div>
              </div>
            </div>
          </div>
          <button
            className=""
            onClick={[Function]}
          >
            <svg
              height="24"
              style={
                Object {
                  "filter": "drop-shadow( 0px 2px 0.5px rgba(0, 0, 0, .3",
                }
              }
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M8 5v14l11-7z"
                fill="#FFFFFF"
              />
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
            </svg>
          </button>
        </div>
        <div
          className=""
        >
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="typo-label2"
            >
              making a project
            </div>
            <p
              className=""
              data-baseweb="typo-p2"
            >
              We needed a view that would allow authors to create project and view them all at a glance. There were many stakeholders all collaborating.  The idea was to enable any of them to rapidly prototype a page and the best way to do this was to clone and version control another project.
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Page Renderer  should render postHeader 1`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <section
    className=""
  >
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className=""
        >
          <div
            className=""
            data-baseweb="typo-label1"
          >
            Web Application
          </div>
          <div
            className=""
            data-baseweb="typo-display2"
          >
            Stanley CMS
          </div>
          <p
            className=""
            data-baseweb="typo-p1"
          >
            A real-time WYSIWYG editor for project owners, designers, copywriters to rapid prototype pages.
          </p>
        </div>
        <div
          className=""
        >
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="typo-label2"
              style={
                Object {
                  "textTransform": "capitalize",
                }
              }
            >
              client
            </div>
            <p
              className=""
              data-baseweb="typo-p4"
            >
              Nike Jordan
            </p>
          </div>
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="typo-label2"
              style={
                Object {
                  "textTransform": "capitalize",
                }
              }
            >
              collaborators
            </div>
            <p
              className=""
              data-baseweb="typo-p4"
            >
              <span>
                Jimmy Soat (Creative Director)
                <br />
              </span>
              <span>
                Stanley Yeung (Visual Designer)
                <br />
              </span>
              <span>
                Jeffrey Qua (Developer)
                <br />
              </span>
            </p>
          </div>
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="typo-label2"
              style={
                Object {
                  "textTransform": "capitalize",
                }
              }
            >
              company
            </div>
            <p
              className=""
              data-baseweb="typo-p4"
            >
              AKQA
            </p>
          </div>
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="typo-label2"
              style={
                Object {
                  "textTransform": "capitalize",
                }
              }
            >
              resources
            </div>
            <p
              className=""
              data-baseweb="typo-p4"
            >
              <span>
                <a
                  className=""
                  data-baseweb="link"
                  href="https://www.github.com"
                >
                  Github
                </a>
                <br />
              </span>
              <span>
                <a
                  className=""
                  data-baseweb="link"
                  href="https://google.com"
                >
                  Design Files
                </a>
                <br />
              </span>
              <span>
                <a
                  className=""
                  data-baseweb="link"
                  href="http://www.google.com"
                >
                  Website
                </a>
                <br />
              </span>
            </p>
          </div>
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="typo-label2"
              style={
                Object {
                  "textTransform": "capitalize",
                }
              }
            >
              role
            </div>
            <p
              className=""
              data-baseweb="typo-p4"
            >
              Developer/Designer
            </p>
          </div>
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="typo-label2"
              style={
                Object {
                  "textTransform": "capitalize",
                }
              }
            >
              technology
            </div>
            <p
              className=""
              data-baseweb="typo-p4"
            >
              <span>
                AngularJS
                <br />
              </span>
              <span>
                Firebase
                <br />
              </span>
            </p>
          </div>
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="typo-label2"
              style={
                Object {
                  "textTransform": "capitalize",
                }
              }
            >
              when
            </div>
            <p
              className=""
              data-baseweb="typo-p4"
            >
              Feb 2014 - Mar 2014 (8 weeks)
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Page Renderer  should render quote 1`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <section
    className=""
  >
    <div
      className=""
    >
      <blockquote
        className=""
      >
        <span
          className=""
        >
          <p
            className=""
            data-baseweb="typo-p1"
          >
            "
            I want to design Nike web pages that are responsive without making a bunch of mocks photoshop.  I want to inline-edit copy and be able to drag and drop images directly from my desktop into the page.”
            "
          </p>
        </span>
        <div
          className=""
        >
          <p
            className=""
            data-baseweb="typo-p4"
          >
            jimmy soat
            <br />
            design director
            <br />
            2015-01-31T18:30:00.000Z
          </p>
        </div>
      </blockquote>
    </div>
  </section>
</div>
`;

exports[`Page Renderer  should render summary 1`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <section
    className=""
  >
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className=""
        >
          <div
            className=""
            data-baseweb="typo-label1"
          >
            00
          </div>
          <div
            className=""
            data-baseweb="typo-display4"
          >
            Summary
          </div>
          <p
            className=""
            data-baseweb="typo-p1"
          >
            The agency AKQA and their client Nike Jordan worked together to complete dozens of champaign webpages. Nike ran the majority of the creative process with their in-house copywriters, photographers, and creative directors. AKQA was the production muscle, bring together all the creative assets into a www.nike.com hosted webpage. Both Nike and AKQA did great work together; however, AKQA’s goal was to have more creative ownership. Unfortunatly for AKQA this ownership was seemingly impossible. The process between AKQA and Nike was fragemented, we were given hard deadlines, short production times, and low budgets. All of which, created a bottleneck that essentially blocked any progress towards more creative ownership. We had a mission to fix this fragemented process by bridging the the communication gap between Client Nike and Agency AKQA.
          </p>
        </div>
        <div
          className=""
        >
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="aspect-ratio-box"
            >
              <div
                className=""
                data-baseweb="aspect-ratio-box-body"
                onClick={[Function]}
              >
                <svg
                  className=""
                  data-baseweb="icon"
                  viewBox="0 0 24 24"
                >
                  <title>
                    Arrow Right
                  </title>
                  <path
                    clipRule="evenodd"
                    d="M6 12C6 12.5523 6.44772 13 7 13H14.5858L12.2929 15.2929C11.9024 15.6834 11.9024 16.3166 12.2929 16.7071C12.6834 17.0976 13.3166 17.0976 13.7071 16.7071L17.7071 12.7071C17.8946 12.5196 18 12.2652 18 12C18 11.7348 17.8946 11.4804 17.7071 11.2929L13.7071 7.29289C13.3166 6.90237 12.6834 6.90237 12.2929 7.29289C11.9024 7.68342 11.9024 8.31658 12.2929 8.70711L14.5858 11H7C6.44772 11 6 11.4477 6 12Z"
                    fillRule="evenodd"
                  />
                </svg>
              </div>
            </div>
            <div
              className=""
              data-baseweb="typo-label4"
            >
              Demo Video
            </div>
          </div>
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="aspect-ratio-box"
            >
              <div
                className=""
                data-baseweb="aspect-ratio-box-body"
                onClick={[Function]}
              >
                <svg
                  className=""
                  data-baseweb="icon"
                  viewBox="0 0 24 24"
                >
                  <title>
                    Arrow Right
                  </title>
                  <path
                    clipRule="evenodd"
                    d="M6 12C6 12.5523 6.44772 13 7 13H14.5858L12.2929 15.2929C11.9024 15.6834 11.9024 16.3166 12.2929 16.7071C12.6834 17.0976 13.3166 17.0976 13.7071 16.7071L17.7071 12.7071C17.8946 12.5196 18 12.2652 18 12C18 11.7348 17.8946 11.4804 17.7071 11.2929L13.7071 7.29289C13.3166 6.90237 12.6834 6.90237 12.2929 7.29289C11.9024 7.68342 11.9024 8.31658 12.2929 8.70711L14.5858 11H7C6.44772 11 6 11.4477 6 12Z"
                    fillRule="evenodd"
                  />
                </svg>
              </div>
            </div>
            <div
              className=""
              data-baseweb="typo-label4"
            >
              Demo Video
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Page Renderer  should render summary 2 1`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <section
    className=""
  >
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className=""
        >
          <div
            className=""
            data-baseweb="typo-label1"
          >
            04
          </div>
          <div
            className=""
            data-baseweb="typo-display4"
          >
            Design Execution
          </div>
          <p
            className=""
            data-baseweb="typo-p1"
          >
            We had to move quick, we went from a list of product requirements to paper prototype with in the first day.  We based most of our interaction design on the precidence of teams’ tooling e.g. Photoshop and CQ5. Next, we gathered all of the creative assets, e.g. fonts, color pallets, icongraphy, etc, in preperation of developing a MVP.
          </p>
        </div>
        <div
          className=""
        >
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="aspect-ratio-box"
            >
              <div
                className=""
                data-baseweb="aspect-ratio-box-body"
                onClick={[Function]}
              >
                <svg
                  className=""
                  data-baseweb="icon"
                  viewBox="0 0 24 24"
                >
                  <title>
                    Arrow Right
                  </title>
                  <path
                    clipRule="evenodd"
                    d="M6 12C6 12.5523 6.44772 13 7 13H14.5858L12.2929 15.2929C11.9024 15.6834 11.9024 16.3166 12.2929 16.7071C12.6834 17.0976 13.3166 17.0976 13.7071 16.7071L17.7071 12.7071C17.8946 12.5196 18 12.2652 18 12C18 11.7348 17.8946 11.4804 17.7071 11.2929L13.7071 7.29289C13.3166 6.90237 12.6834 6.90237 12.2929 7.29289C11.9024 7.68342 11.9024 8.31658 12.2929 8.70711L14.5858 11H7C6.44772 11 6 11.4477 6 12Z"
                    fillRule="evenodd"
                  />
                </svg>
              </div>
            </div>
            <div
              className=""
              data-baseweb="typo-label4"
            >
              Demo Video
            </div>
          </div>
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="aspect-ratio-box"
            >
              <div
                className=""
                data-baseweb="aspect-ratio-box-body"
                onClick={[Function]}
              >
                <svg
                  className=""
                  data-baseweb="icon"
                  viewBox="0 0 24 24"
                >
                  <title>
                    Arrow Right
                  </title>
                  <path
                    clipRule="evenodd"
                    d="M6 12C6 12.5523 6.44772 13 7 13H14.5858L12.2929 15.2929C11.9024 15.6834 11.9024 16.3166 12.2929 16.7071C12.6834 17.0976 13.3166 17.0976 13.7071 16.7071L17.7071 12.7071C17.8946 12.5196 18 12.2652 18 12C18 11.7348 17.8946 11.4804 17.7071 11.2929L13.7071 7.29289C13.3166 6.90237 12.6834 6.90237 12.2929 7.29289C11.9024 7.68342 11.9024 8.31658 12.2929 8.70711L14.5858 11H7C6.44772 11 6 11.4477 6 12Z"
                    fillRule="evenodd"
                  />
                </svg>
              </div>
            </div>
            <div
              className=""
              data-baseweb="typo-label4"
            >
              Demo Video
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Page Renderer  should render summary 3 1`] = `
<div
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <section
    className=""
  >
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className=""
        >
          <div
            className=""
            data-baseweb="typo-label1"
          />
          <div
            className=""
            data-baseweb="typo-display4"
          />
          <p
            className=""
            data-baseweb="typo-p1"
          />
        </div>
        <div
          className=""
        >
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="aspect-ratio-box"
            >
              <div
                className=""
                data-baseweb="aspect-ratio-box-body"
                onClick={[Function]}
              >
                <svg
                  className=""
                  data-baseweb="icon"
                  viewBox="0 0 24 24"
                >
                  <title>
                    Arrow Right
                  </title>
                  <path
                    clipRule="evenodd"
                    d="M6 12C6 12.5523 6.44772 13 7 13H14.5858L12.2929 15.2929C11.9024 15.6834 11.9024 16.3166 12.2929 16.7071C12.6834 17.0976 13.3166 17.0976 13.7071 16.7071L17.7071 12.7071C17.8946 12.5196 18 12.2652 18 12C18 11.7348 17.8946 11.4804 17.7071 11.2929L13.7071 7.29289C13.3166 6.90237 12.6834 6.90237 12.2929 7.29289C11.9024 7.68342 11.9024 8.31658 12.2929 8.70711L14.5858 11H7C6.44772 11 6 11.4477 6 12Z"
                    fillRule="evenodd"
                  />
                </svg>
              </div>
            </div>
            <div
              className=""
              data-baseweb="typo-label4"
            >
              Demo Video
            </div>
          </div>
          <div
            className=""
          >
            <div
              className=""
              data-baseweb="aspect-ratio-box"
            >
              <div
                className=""
                data-baseweb="aspect-ratio-box-body"
                onClick={[Function]}
              >
                <svg
                  className=""
                  data-baseweb="icon"
                  viewBox="0 0 24 24"
                >
                  <title>
                    Arrow Right
                  </title>
                  <path
                    clipRule="evenodd"
                    d="M6 12C6 12.5523 6.44772 13 7 13H14.5858L12.2929 15.2929C11.9024 15.6834 11.9024 16.3166 12.2929 16.7071C12.6834 17.0976 13.3166 17.0976 13.7071 16.7071L17.7071 12.7071C17.8946 12.5196 18 12.2652 18 12C18 11.7348 17.8946 11.4804 17.7071 11.2929L13.7071 7.29289C13.3166 6.90237 12.6834 6.90237 12.2929 7.29289C11.9024 7.68342 11.9024 8.31658 12.2929 8.70711L14.5858 11H7C6.44772 11 6 11.4477 6 12Z"
                    fillRule="evenodd"
                  />
                </svg>
              </div>
            </div>
            <div
              className=""
              data-baseweb="typo-label4"
            >
              Demo Video
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
